### PR TITLE
Spinner added!

### PR DESCRIPTION
**Pull request by DineshKoushik**
## Related Issue 

- Info about the related issue 
- (The issue was to add a spinner gif in place of "initializing data".)

Closes: #[issue number that will be closed through this PR]

### Describe the changes you've made

Give a clear description what modifications you have made
(I added a simple bordered spinner in place of "initializing data".)

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Enhancement in the existing features
- [ ] Other

## Mention any unusual behaviour of your code (Write NA if not)
Any unusual behaviour of your code
NA

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly where ever it was hard to understand.
- [x] My changes generate no new warnings.
- [x] I have verified/tested my code by running it locally.

## Additional Info (optional)
Any additional information you want to give

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,11 +7,13 @@
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
     "axios": ">=0.21.1",
+    "bootstrap": "^4.6.0",
     "bootstrap-4-react": "0.0.59",
     "bootstrap4": "0.0.1-security",
     "concurrently": "^6.0.0",
     "nodemailer": "^6.4.11",
     "react": "^16.13.1",
+    "react-bootstrap": "^1.5.2",
```
Comment (Namanl2001) [inline package.json:16]: We have used `bootstrap-4-react` in this project so could you mind using that only for adding the spinner. Adding `bootstrap` and `react-bootstrap` don't make sense IMO.
Also make the spinner to `center`.

Comment (DineshKoushik) [reply package.json:16]: Alright!

**Comment by DineshKoushik** [comment]
@Namanl2001 Can you check if this works?

**Comment by Namanl2001** [comment]
have you checked your changes for mobile phone? 
Also run yarn install in backend as well as in root dir, if there are any changes in lock filee please commit and push them.

File: package.json (hunk 1 above)
Comment (DineshKoushik) [reply package.json]: @Namanl2001 `bootstrap-4-react`  doesn't have css files, does it? Can I know how to import spinner if possible?

Comment (Namanl2001) [reply package.json]: @sagar-barapatre / @guptaji6489  Could help you.

Comment (sagar-barapatre) [reply package.json]: Have a look [here](https://react-bootstrap.github.io/components/spinners/)

Comment (DineshKoushik) [reply package.json]: @sagar-barapatre , I mean there are no css files in node_modules/bootstrap-4-react

**Comment by Namanl2001** [comment]
Any update?

**Comment by Namanl2001** [comment]
Closing due to no response

**Pull request closed without merging**